.github/workflows/upload_app_store.yml

```diff
@@ -0,0 +1,62 @@
+name: Upload IPA to App Store
+on: workflow_dispatch
+jobs:
+  check_branch:
+    name: "Check branch ('release*' or 'main')."
+    runs-on: macos-11
+    steps:
+      - id: early_exit
+        if: ${{ github.ref != 'refs/heads/main' && startsWith(github.ref, 'refs/heads/release') != true }}
+        name: Early exit failure if not main or release branch.
+        run: exit 1
+  build_test_upload:
+    name: Build, test, and upload to App Store
+    needs: check_branch
+    runs-on: macos-11
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+            node-version: 16
+            cache: "npm"
+            cache-dependency-path: "**/package-lock.json"
+      - name: Build core and features
+        run: ./build.js
+        env:
+            POLYPOD_POLYPEDIA_REPORT_URL: ${{ secrets.POLYPOD_POLYPEDIA_REPORT_URL }}
+            POLYPOD_POLYPEDIA_REPORT_AUTHORIZATION: ${{ secrets.POLYPOD_POLYPEDIA_REPORT_AUTHORIZATION }}
+      - name: Lock xcode version
+        run: sudo xcode-select -s "/Applications/Xcode_13.2.1.app"
```

Comment:
not a big deal, but here:

https://github.com/polypoly-eu/polyPod/blob/3f9abbc1effb87a7ea5ae46a0615c8effd5c0df0/.github/workflows/ios-test.yml#L28

we're using a different version. We might want to use the same, and set it at the repo secret level.

Reply:
Yes, we should migrate `ios-test` workflow to also use Xcode 13.2.1. Having the Xcode version as a secret and shared between workflows is a really good idea 👍.

Reply:
... which I don't have privileges to do ... If you decide to do that, tell me and I'll change the default test to use it.

Reply:
Me neither, @fhd should have the necessary rights.

Reply:
> ... which I don't have privileges to do ... If you decide to do that, tell me and I'll change the default test to use it.

Which is a reason why it might not be a good idea. Alternatively, let me see if there's some target OS declaration somewhere that we can extract via awk/Perl/sed/whatever; plan C would be to have a `xcode-version` file that we can simply read.

Reply:
Yep, there's this thing:

```
PolyPodApp/PolyPod.xcodeproj/project.pbxproj:                           IPHONEOS_DEPLOYMENT_TARGET = 13.2;
```

Let me see if we can use that.

Reply:
> Me neither, @fhd should have the necessary rights.

I guess we'll try to use some other shared variable then. Let me try the thing above.

Reply:
I don't think that OS version can be used since it is not directly related to the Xcode version. For example in the very near future we will most likely update `IPHONEOS_DEPLOYMENT_TARGET` to 14.0. 
It may be misleading at the moment `IPHONEOS_DEPLOYMENT_TARGET = 13.2` is somewhat similar to `Xcode 13.2.1`, but it is just a coincidence.

Reply:
True. However, I seem to remember there was a table that listed the minimum Xcode version for each iOS version.
At any rate, we can then use plan C, put that in a specific file.

Reply:
Yes, each iOS version requires a min Xcode version. But `IPHONEOS_DEPLOYMENT_TARGET` is the minimal supported version. We indeed could derive Xcode version from latest iOS version, but I am not aware of a way to know the latest iOS version from the project config.

Reply:
Conf file it is then.

Reply:
Another option would be to add it to the Makefile, which is already there, and call this from the makefile. Let me try that.